### PR TITLE
Glider cals

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Review by rtravis-whoi — APPROVED** [review]

File: calibration/CTDGVM/CGINS-CTDGVM-00073__20220820.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+73,CC_calibration_date,1660953600000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/CTDGVM/CGINS-CTDGVM-00073__20220820.csv:2]: checked against vendor doc and milliseconds converter. looks good.

File: calibration/CTDGVM/CGINS-CTDGVM-09059__20220817.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9059,CC_calibration_date,1660694400000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/CTDGVM/CGINS-CTDGVM-09059__20220817.csv:2]: checked against vendor doc and milliseconds converter. looks good.

File: calibration/CTDGVM/CGINS-CTDGVM-09295__20220809.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9295,CC_calibration_date,1660003200000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/CTDGVM/CGINS-CTDGVM-09295__20220809.csv:2]: checked against vendor doc and milliseconds converter. looks good.

File: calibration/CTDGVM/CGINS-CTDGVM-09363__20220828.csv [hunk 4]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9363,CC_calibration_date,1661644800000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/CTDGVM/CGINS-CTDGVM-09363__20220828.csv:2]: checked against vendor doc and milliseconds converter. looks good.

File: calibration/PARADM/CGINS-PARADM-50151__20220808.csv [hunk 5]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50151,CC_bsipar_par_scaling,1,
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/PARADM/CGINS-PARADM-50151__20220808.csv:2]: checked against vendor cal cert. looks good.

File: calibration/PARADM/CGINS-PARADM-50202__20220726.csv [hunk 6]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50202,CC_bsipar_par_scaling,1,
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/PARADM/CGINS-PARADM-50202__20220726.csv:2]: checked against vendor cal cert. looks good.

File: calibration/PARADM/CGINS-PARADM-50203__20220812.csv [hunk 7]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50203,CC_bsipar_par_scaling,1,
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/PARADM/CGINS-PARADM-50203__20220812.csv:2]: checked against vendor cal cert. looks good.

File: calibration/PARADM/CGINS-PARADM-50238__20220808.csv [hunk 8]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50238,CC_bsipar_par_scaling,1,
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/PARADM/CGINS-PARADM-50238__20220808.csv:2]: checked against vendor cal cert. looks good.

File: calibration/PARADM/CGINS-PARADM-50244__20220808.csv [hunk 9]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50244,CC_bsipar_par_scaling,1,
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/PARADM/CGINS-PARADM-50244__20220808.csv:2]: checked against vendor cal cert. looks good.

File: calibration/FLORTM/CGINS-FLORTM-03185__20220808.csv [hunk 10]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+3185,CC_angular_resolution,1.076,
+3185,CC_depolarization_ratio,0.039,
+3185,CC_measurement_wavelength,700,
+3185,CC_scattering_angle,124,
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/FLORTM/CGINS-FLORTM-03185__20220808.csv:5]: date and sn good. values check against CGINS-FLORTM-03190__20200423.csv - all look good.

File: calibration/FLORTM/CGINS-FLORTM-03190__20220804.csv [hunk 11]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+3190,CC_angular_resolution,1.076,
+3190,CC_depolarization_ratio,0.039,
+3190,CC_measurement_wavelength,700,
+3190,CC_scattering_angle,124,
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/FLORTM/CGINS-FLORTM-03190__20220804.csv:5]: date and sn good. values check against CGINS-FLORTM-03190__20200423.csv - all look good.

File: calibration/FLORTM/CGINS-FLORTM-04089__20220805.csv [hunk 12]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4089,CC_angular_resolution,1.076,
+4089,CC_depolarization_ratio,0.039,
+4089,CC_measurement_wavelength,700,
+4089,CC_scattering_angle,124,
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/FLORTM/CGINS-FLORTM-04089__20220805.csv:5]: date and sn good. values check against CGINS-FLORTM-03190__20200423.csv - all look good.

File: calibration/FLORTM/CGINS-FLORTM-04467__20220805.csv [hunk 13]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4467,CC_angular_resolution,1.076,
+4467,CC_depolarization_ratio,0.039,
+4467,CC_measurement_wavelength,700,
+4467,CC_scattering_angle,124,
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/FLORTM/CGINS-FLORTM-04467__20220805.csv:5]: date and sn good. values check against CGINS-FLORTM-03190__20200423.csv - all look good.

File: calibration/FLORTM/CGINS-FLORTM-04471__20220808.csv [hunk 14]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+4471,CC_angular_resolution,1.076,
+4471,CC_depolarization_ratio,0.039,
+4471,CC_measurement_wavelength,700,
+4471,CC_scattering_angle,124,
```
[the remark this answers is not in the document]
Comment (sauuyer) [reply calibration/FLORTM/CGINS-FLORTM-04471__20220808.csv:5]: date and sn good. values check against CGINS-FLORTM-03190__20200423.csv - all look good.

**Review by sauuyer — COMMENTED** [review]
all commits checked. no changes needed.

**Review by sauuyer — APPROVED** [review]

**Pull request merged**